packages/dom/src/index.js

```diff
@@ -13,18 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-/**
- * Fires when dom is ready.
- *
- * @param {Function} callback Callback function.
- */
-function domReady(callback) {
-  if ('loading' === document.readyState) {
-    document.addEventListener('DOMContentLoaded', callback);
-  } else {
-    callback();
-  }
-}
-
-export default domReady;
+export { default as stripHTML } from './stripHTML';
+export { default as escapeHTML } from './escapeHTML';
+export { default as getCaretCharacterOffsetWithin } from './getCaretCharacterOffsetWithin';
```

Comment:
This one might be a candidate for the rich-text package since it's used for text editing